makeflow/src/dag_file.c

```diff
@@ -12,14 +12,20 @@ See the file COPYING for details.
 
 #include <stdlib.h>
 
+#define MAKEFLOW_DEFAULT_FILE_SIZE 1024
+////*1024*1024
```

Comment:
Is this in MB?  In any case, please add a comment indicating the units.
Also, remove the comment on line 16.